pom.xml

```diff
@@ -117,10 +122,20 @@
 					<sources>
 						<source>src/main/resources/schema/InExchangeInvoiceStatusSchema.xsd</source>
 					</sources>
-					<packageName>generated.se.inexchange</packageName>
+					<packageName>se.inexchange.generated</packageName>
```

Comment:
I changed this as I though it was cleaner to have all generated classes under the same root package (generated). Don't you agree?